test/test_files/patcher_data/prime1/prime1_refills/world_1.json

```diff
@@ -65,9 +65,13 @@
             "wave": false,
             "plasma": false,
             "missiles": 0,
+            "missileLauncher": true,
```

Comment:
can you modify the rdvgame for this file to have shuffled required mains + shuffled unlimited ammo?